README.md

```diff
@@ -1,4 +1,20 @@
-# advocacy.mozilla.org
+# Overview
+
+This is the source code for [advocacy.mozilla.org]()
+
+Currently there are three parts to the site.
```

Comment:
Good to date this (like "as of April 2016, there are.."), this also makes it easier to flag a need to update the README